Source/Model/TCCProfile.swift

```diff
@@ -167,8 +167,8 @@ struct TCCProfile : Codable {
                               services: services)
         self.version = 1
         self.uuid = UUID().uuidString
-        self.type = content.type
-        self.scope = "system"
+        self.type = "Configuration"
```

Comment:
It looks like the content's type is being set to `com.apple.TCC.configuration-profile-policy` on line 164. Is it safe to override that here to always be `Configuration` instead? Do we know why it was set to `com.apple.TCC.configuration-profile-policy` originally? Looking at [Rich Trouton's repo of example PPPC profiles](https://github.com/rtrouton/privacy_preferences_control_profiles), they have `Configuration` for the type (at least the ones I looked at).

Reply:
It looks like "Configuration" is correct according to the spec:
https://developer.apple.com/business/documentation/Configuration-Profile-Reference.pdf

"The only supported value is Configuration."

However, we should change the value in the section that creates the content variable.

Reply:
Good find. In that case, I'm going to mark as needs work to address this.

Reply:
Line 170 isn't overriding line 164, it's just not reusing it anymore. We want self.type to be Configuration, then we want self.content[].type, which is set on line 176 with the content variable, to be com.apple.TCC.configuration-profile-policy according to spec: https://developer.apple.com/documentation/devicemanagement/privacypreferencespolicycontrol

So if we change line 164 it would actually break the nested level. The changes to 170 correctly set the top level to Configuration and leaves the nested level payload type com.apple.TCC.configuration-profile-policy.

Let me know if that makes sense or not, I'm pretty confident it's correct as-is.

Reply:
@pirkla that makes sense now, sorry for the confusion

Reply:
Ah you're right @pirkla, I was reading it incorrectly. 👍